dumb-jump.el

```diff
@@ -2111,6 +2108,13 @@ to keep looking for another root."
         (thing-at-point 'symbol)
       (thing-at-point 'symbol t))))
 
+(defun dumb-jump--get-symbol-start ()
+  "Get bounds of symbol at point"
+  (- (if (region-active-p)
+         (region-beginning)
+       (car (bounds-of-thing-at-point 'symbol)))
+     (point-at-bol)))
```

Comment:
`point-at-bol` is a XEmacs compatibility function, I think it would be better to use `line-beginning-position` instead.